domains/authorization/Authorization.API/API/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -80,6 +80,14 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .IsRequired()
                         .HasColumnType("text");
 
+                    b.Property<DateTimeOffset?>("ServiceProviderTermsAcceptanceDate")
+                        .HasColumnType("timestamp with time zone");
+
+                    b.Property<bool>("ServiceProviderTermsAccepted")
+                        .ValueGeneratedOnAdd()

```

Comment:
Ah, sorry... the comment should have been in applicationDbContext. Since _ServiceProviderTermsAccepted_ should default to _false_, and column has a default value everything looks good.

Reply:
HAHA :D Not only me who comments on generated code :P